modules/setting/incoming_email.go

```diff
@@ -44,7 +44,7 @@ func loadIncomingEmailFrom(rootCfg ConfigProvider) {
 }
 
 func checkReplyToAddress(address string) error {
-	parsed, err := mail.ParseAddress(IncomingEmail.ReplyToAddress)
+	parsed, err := mail.ParseAddress(address)
```

Comment:
I think this is unrelated and should be another PR. Line 56 & Line 65 also used this global variable.

Reply:
Updated in 2eb79b2e1a4d4d4dd02888d6a34c63c011234e33